Changelog.md

```diff
@@ -11,6 +11,7 @@ Compiler Features:
  * Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips CBOR metadata from getting appended at the end of the bytecode.
  * Yul EVM Code Transform: Generate more optimal code for user-defined functions that always terminate a transaction. No return labels will be pushed for calls to functions that always terminate.
  * Yul Optimizer: Allow replacing the previously hard-coded cleanup sequence by specifying custom steps after a colon delimiter (``:``) in the sequence string.
+ * Yul Optimizer: Eliminate `keccak256`` calls if the value was already calculated by a previous call and can be reused.
```

Comment:
```suggestion
 * Yul Optimizer: Eliminate ``keccak256`` calls if the value was already calculated by a previous call and can be reused.
```

Reply:
Ugh, damn Clion keeps adding double quotes, and then I have to delete one every time.

Reply:
We should consistently define whether using double or single ones in general - it's a bit of a mess right now :-). Didn't this even come up before already :-)?

Reply:
Luckily, we don't keep a record of such glorious mechanisms :-). Or wait, I see that we indeed do...

**Use double backticks in RST (``` ``x`` ```). Prefer single backticks in Markdown (`` `x` ``),**
but note that double backticks are valid too and we use them in some cases for legacy reasons.

So yeah, use either one or both - extremely helpful that we cleared that up :-D.

Reply:
Should we change that to "always use single backticks in Markdown" and convert all existing .md files?